Gemfile.lock

```diff
@@ -38,16 +38,16 @@ GIT
       rest-client (~> 1.8)
 
 GIT
-  remote: git://github.com/lostisland/faraday_middleware.git
+  remote: https://github.com/lostisland/faraday_middleware.git
   revision: 59088da02940d0ee2010b2e3156343346767c31e
   branch: master
   specs:
     faraday_middleware (0.10.0)
       faraday (>= 0.7.4, < 1.0)
 
 GIT
-  remote: git://github.com/tumblr/tumblr_client.git
-  revision: 0c59b04e49f2a8c89860613b18cf4e8f978d8dc7
```

Comment:
Can you lock the ref to `0c59b04e49f2a8c89860613b18cf4e8f978d8dc7` in the Gemfile?